zap/src/main/java/org/parosproxy/paros/core/scanner/AbstractPlugin.java

```diff
@@ -587,6 +588,62 @@ protected boolean isFileExist(HttpMessage msg) {
         return parent.getAnalyser().isFileExist(msg);
     }
 
+    /**
+     * Tells whether or not the message matches the specific {@code CustomPageType}
+     *
+     * @param msg the message that will be checked
+     * @param cpType the custom page type to be checked
+     * @return {@code true} if the message matches, {@code false} otherwise
+     * @since TODO Add version
+     */
+    private boolean isCustomPage(HttpMessage msg, CustomPage.Type cpType) {
+        return parent.isCustomPage(msg, cpType);
+    }
+
+    /**
+     * Tells whether or not the message matches {@code CustomPageType.OK_200} definitions.
+     *
+     * @param msg the message that will be checked
+     * @return {@code true} if the message matches, {@code false} otherwise
+     * @since TODO Add version
+     */
+    protected boolean isPage200(HttpMessage msg) {
```

Comment:
The `ScriptsActiveScanner` needs to override these methods and change to public to allow the scripts to use them.